### PR TITLE
Use 1+2 keys for toggling before+after on /analyzer

### LINKED ISSUE #1264: Reftest analyzer key bindings
Mozilla's reftest analyzer has keys "1" and "2" bound to the before and after images, respectively. That makes it very easy to toggle back and forth between the two images, and also makes that possible while keeping your pointer over a part of the image you're interested in. (I.e. the zoomed view should update on toggling.) It would be helpful if the WPT reftest analyzer could have the same behavior.

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Fixes #1264 

## Review Information
Also drive-by fixes the width when the error message is shown, and handling the spinner stopping onerror.

### CODE REVIEW

File: webapp/templates/analyzer.html [hunk 1]
```diff
@@ -2,11 +2,23 @@
 <html>
 <head>
 {{ template "_head_common.html" }}
-<script type="module" src="/components/reftest-analyzer.js"></script>
+<script type="module">
+import '/components/reftest-analyzer.js';
+
+document.body.addEventListener('keydown', e => {
+  if (e.which == 49) {
```
Comment (Hexcles) [inline webapp/templates/analyzer.html]: Let's add a comment to say they are numkey '1' and '2'.

File: webapp/components/reftest-analyzer.js [hunk 2]
```diff
@@ -101,9 +104,11 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
         </div>
```
Comment (Hexcles) [inline webapp/components/reftest-analyzer.js]: Optional: add tooltips to the two radio buttons to show their keyboard bindings.

Comment (lukebjerring) [reply webapp/components/reftest-analyzer.js]: I'll avoid this for now as the binding behaviour is added in the page (body keydown), not the component.

**Comment by lukebjerring** [comment]
Merging since comments didn't break anything.

**Pull request merged**